config/main.py

```diff
@@ -4524,8 +4529,9 @@ def validate_vlan_format(text):
 @click.argument('interface_name', metavar='<interface_name>', required=True)
 @click.argument("ip_addr", metavar="<ip_addr>", required=True)
 @click.argument('gw', metavar='<default gateway IP address>', required=False)
+@click.option('-i','--ignore-vlan',is_flag=True, required=False, help="Allow configuration on not created Vlan")
```

Comment:
I don't think this must be handled in this way. Lets simply throw an error if vlan does not exist asking user to create Vlan first?

Reply:
done